tests/integrationtest/r/executor/infoschema_reader.result

```diff
@@ -283,3 +283,34 @@ SELECT * FROM information_schema.user_privileges WHERE grantee="'usageuser'@'%'"
 GRANTEE	TABLE_CATALOG	PRIVILEGE_TYPE	IS_GRANTABLE
 'usageuser'@'%'	def	BACKUP_ADMIN	NO
 'usageuser'@'%'	def	SELECT	YES
+select VARIABLE_NAME from information_schema.VARIABLES_INFO where DEFAULT_VALUE = CURRENT_VALUE and variable_name in ('tidb_txn_mode', 'tidb_enable_async_commit','tidb_enable_1pc', 'tidb_mem_oom_action', 'tidb_enable_auto_analyze', 'tidb_row_format_version', 'tidb_txn_assertion_level', 'tidb_enable_mutation_checker', 'tidb_pessimistic_txn_fair_locking') order by VARIABLE_NAME;
+VARIABLE_NAME
+tidb_enable_1pc
+tidb_enable_async_commit
+tidb_enable_auto_analyze
+tidb_enable_mutation_checker
+tidb_mem_oom_action
+tidb_pessimistic_txn_fair_locking
+tidb_row_format_version
+tidb_txn_assertion_level
+tidb_txn_mode
```

Comment:
`tidb_txn_mode` already covered by https://github.com/pingcap/tidb/pull/48500, no need to add it here.

Reply:
> `tidb_txn_mode` already covered by #48500, no need to add it here.

done. https://github.com/pingcap/tidb/pull/49524/commits/0558302f2ac1a9cc083a7112b6cd51f13b7aed77